src/clr-angular/data/tree-view/_tree-view.clarity.scss

```diff
@@ -148,17 +148,16 @@
     }
   }
 
-  .clr-treenode-checkbox {
+  .clr-tree-node-content-container > .clr-checkbox-wrapper {
     //Dimensions
     height: $clr-tree-node-touch-target;
     width: $clr-tree-node-touch-target;
     padding-top: $clr-tree-checkbox-padding-top;
     padding-left: $clr-tree-checkbox-padding-left;
+  }
 
-    //States
-    &:first-child {
-      margin-left: $clr-tree-node-touch-target;
-    }
+  .clr-tree-node-content-container > .clr-checkbox-wrapper:first-child {
```

Comment:
I know we want to keep nesting down but re-declaring the selectors but this seems like something un-necesarily duplicated.

Reply:
It was necessary to target with the > selector or it was not specific enough.